src/services/EdFi.DataManagementService.Api/Configuration/AppSettings.cs

```diff
@@ -3,12 +3,23 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-namespace EdFi.DataManagementService.Api.Configuration
+using Microsoft.Extensions.Options;
+
+namespace EdFi.DataManagementService.Api.Configuration;
+
+public class AppSettings
+{
+    public int BeginAllowedSchoolYear { get; set; }
+    public int EndAllowedSchoolYear { get; set; }
+    public required string AuthenticationService { get; set; }
+}
+
+public class AppSettingsValidator : IValidateOptions<AppSettings>
 {
-    public class AppSettings
+    public ValidateOptionsResult Validate(string? name, AppSettings options)
     {
-        public int BeginAllowedSchoolYear { get; set; }
-        public int EndAllowedSchoolYear { get; set; }
-        public required string AuthenticationService { get; set; }
+        return options.AuthenticationService == null
```

Comment:
Suggest changing to detect empty string or spaces, in addition to null.

Reply:
Done.